app/views/facility_accounts/_account_balances.html.haml

```diff
@@ -7,14 +7,13 @@
       %tr
         %th= t(".th.account")
         %th= t(".th.owner")
-        %th.currency= t(".th.total")
+        %th= t(".th.expiration")
     %tbody
       - @accounts.each do |account|
         %tr
           %td= payment_source_link_or_text(account)
           %td= account.owner_user.full_name if account.owner_user
-          %td.currency
-            = number_to_currency(account.unreconciled_total(current_facility))
+          %td= human_date(account.expires_at)
```

Comment:
Don't know if nows the time to do it, but could `humam_date`'s implementation be replaced with `date.strftime("%B %e, %Y")` or move that format into `en.yml` and use Rail's `l` helper?